code/test/TemplateStudioForWinUICs.Tests/TestData/WinUI/SC/StyleCop/Param_ProjectName/GlobalSuppressions.cs

```diff
@@ -23,3 +23,4 @@
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1602:Enumeration items should be documented", Justification = "This is a newly added analyzer check--suppressing as so much existing template code fails this rule.")]
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1633:File must have header", Justification = "By default we do not include a header in the geenrated files.")]
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1652:Enable XML documentation output", Justification = "We leave this as a choice for the developer to enable. As the generated code doesn't include XML comments, enabling this would just create more issues with the generated code.")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1401:FieldsMustBePrivate", Justification = "Sometimes, it makes sense to have public member variables.")]
```

Comment:
Revert this. Public members should be properties.

Reply:
it is a temporary change. but change to property is simple enough. i can do that